client/app/_consts/iconColor.ts

```diff
@@ -37,3 +37,15 @@ export const iconColor = [
     value: 'WHITE',
   },
 ] as const
```

Comment:
Could you delete this iconColor variable above since we no longer use it?

Reply:
Deleted it!
624367c60171475ad857a89db52de0b4a79c1b56

Reply:
Didn't see the error below. I fixed it. Thanks! 
624367c60171475ad857a89db52de0b4a79c1b56